kernels/set_plane.cl

```diff
@@ -0,0 +1,14 @@
+__constant sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
+
+__kernel void set_plane(
+    IMAGE_dst_TYPE  dst,
+    const int       cst,
+    const float     scalar
+)
+{
+  const int x = get_global_id(0);
+  const int y = get_global_id(1);
+  const int z = cst;
```

Comment:
```suggestion
    const int       plane,
    const float     scalar
)
{
  const int x = get_global_id(0);
  const int y = get_global_id(1);
  const int z = plane;
```

Reply:
`cst` stand for constant, we can use something else. I would suggest `index`, avoiding the trinity `column`, `slice`, `row` tags.